src/transformState.js

```diff
@@ -62,7 +62,28 @@
  * @param {Object[]} transforms
  */
 function transformState(state, transforms) {
-  // write code here
+  for (const transform of transforms) {
+    switch (transform.operation) {
+      case 'addProperties' :
+        Object.assign(state, transform.properties);
+        break;
+
+      case 'removeProperties' :
+        for (const property of transform.properties) {
+          delete state[property];
+        }
+        break;
+
+      case 'clear':
+        for (const property in state) {
+          delete state[property];
+        }
+        break;
+
+      default:
+        break;
```

Comment:
Good and pretty laconic solution, but this part of code is unnecessary because `case 'clear'` is the last option and  we don't have to do something with our object by default.

Reply:
Thanks! Did this 'На Автомате') Пофиксил)